src/Dropdown.vue

```diff
@@ -48,15 +46,22 @@ export default {
     },
     type: {
       type: String,
-      default: 'default'
+      default: ''
```

Comment:
We should choose one of the official button types from Bootstrap 4 (probably "light") and make it the default. Any specific styling (such as border), we should override below in the Vue component.

Reply:
Continuation of discussion: We agree to override `.btn-light` class in `markbind.css`, as a separate PR.